angular-fluidVid.js

```diff
@@ -27,7 +27,11 @@ angular.module('tinacious.fluidVid', [])
             case 'vimeo':
               videoCode = '<iframe src="http://player.vimeo.com/video/' + videoId + '" frameborder="0" webkitAllowFullScreen mozallowfullscreen allowFullScreen></iframe>';
               break;
-            
+
+            case 'facebook':
+              videoCode = '<iframe src="https://www.facebook.com/plugins/video.php?href=' + videoId + '" width="400" height="400" style="border:none;overflow:hidden" scrolling="no" frameborder="0" allowTransparency="true" allowFullScreen="true"></iframe>';
```

Comment:
The existing code style should be respected re: attribute value assignment for the iframe, e.g. `allowFullscreen` rather than `allowFullscreen="true"`. It is generally best practice to follow the existing file's code style when contributing to an open-source project.

You have also added inline styles—`border: none;` is a style choice rather than required for the video to render. Why was `overflow: hidden` added? If neither of these are required for the functioning of Facebook video, please remove, otherwise add some spaces to make the declarations more readable.

Also, please remove the counter-productive width and height attributes—these are overwritten by the JavaScript code, and the whole point of this plugin is to not have hard video dimensions. These are redundant and can be confusing to anyone else reading the code because there is no point in adding attributes that get overridden.